client/src/pages/ChooseGameModePage.js

```diff
@@ -1,43 +1,32 @@
 import React from 'react';
 import styled from '@emotion/styled';
 import PropTypes from 'prop-types';
-import Header from '../components/Header';
-import Footer from '../components/Footer';
 import Input from '../components/Input';
 import Planet from '../icons/Planet';
 import Lock from '../icons/Lock';
 import ButtonLink from '../components/ButtonLink';
 import SettingsButton from '../components/SettingsButton';
 import Rocket from '../icons/Rocket';
+import { flexColumnCenter, linearGradientBoxShadow } from '../styles/General';
+import TextWrapper from '../components/TextWrapper';
 
 const Main = styled.main`
-  display: flex;
+  ${flexColumnCenter};
   justify-content: space-around;
-  align-items: center;
-  flex-flow: column nowrap;
   height: 100vh;
   width: 100vw;
   overflow: scroll;
 `;
 const Container = styled.div`
-  display: flex;
-  flex-flow: column;
+  ${flexColumnCenter};
+  ${linearGradientBoxShadow};
   justify-content: space-around;
-  align-items: center;
   align-self: center;
-  width: 340px;
-  height: 50%;
-
-  background: linear-gradient(
-    to right,
-    ${props => props.theme.colors.card2},
-    ${props => props.theme.colors.card1}
-  );
-  border-radius: 25px 0px 25px 0px;
-  box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.16);
   text-align: center;
-  align-self: center;
   color: ${props => props.theme.colors.text1};
+  width: 340px;
```

Comment:
again, you should consider using `%` as unit here for easy "responsiveness"

Reply:
see above. but thanks again!